cleansio/wordlist/user/__init__.py

```diff
@@ -0,0 +1,4 @@
+""" Runs after every import """
+
+from .user_word_list_loader import UserWordListLoader
```

Comment:
Can you please use Explicit instead of Word to be consistent with Levin?

Reply:
CHanged